WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleAndDetailTableViewCell.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+
+final class TitleAndSubtitleAndDetailTableViewCell: UITableViewCell, SearchResultCell {
+    typealias SearchModel = ViewModel
+
+    @IBOutlet private var subtitleLabel: UILabel!
+    @IBOutlet private var titleLabel: UILabel!
+
+    static func register(for tableView: UITableView) {
+        tableView.registerNib(for: self)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureBackground()
+    }
+
+    func configureCell(searchModel: ViewModel) {
+        configureCell(viewModel: searchModel)
+        subtitleLabel.applyStrongCaption1Style()
+    }
+
+    func configureCell(viewModel: ViewModel) {
+        subtitleLabel.text = viewModel.subtitle
+        accessibilityLabel = viewModel.accessibilityLabel
+        setupTitleLabelText(with: viewModel)
+    }
```

Comment:
Question: Could we collapse this setup into one method? 
```suggestion
    func configureCell(searchModel: ViewModel) {
        subtitleLabel.applyStrongCaption1Style()
        subtitleLabel.text = viewModel.subtitle
        accessibilityLabel = viewModel.accessibilityLabel
        setupTitleLabelText(with: viewModel)
    }
```

Otherwise we can make the second `configureCell` private, since we only have to expose one for protocol conformance.

Reply:
Good catch, done in 91ffc601ce65fb2d4c635dc247e2880364cc36b5